charts/vector/templates/NOTES.txt

```diff
@@ -12,7 +12,7 @@ Vector is starting in your cluster. After a few minutes, you can use Vector's AP
   {{- else if .Values.customConfig.api.enabled }}
 Vector is starting in your cluster. After a few minutes, you can use Vector's API to view internal metrics by running:
 
-	kubectl -n {{ .Release.Namespace }} exec -it statefulset/{{ include "vector.fullname" . }} -- vector top --url {{ .Values.customConfig.api.address | default "http://127.0.0.1:8686/graphql" }}
+	kubectl -n {{ .Release.Namespace }} exec -it statefulset/{{ include "vector.fullname" . }} -- vector top --url {{ printf "http://%s/graphql" .Values.customConfig.api.address | default "http://127.0.0.1:8686/graphql" }}
```

Comment:
This command would fail if the `api.address` was replaced in `customConfig` as we didn't pass the full url and path